### DISCUSSION TITLE
Support for Panel and Dashboard JSON in YAML format

### DISCUSSION THREAD

**Discussion by chrisharbro** [grafana/grafana#69190] [Dashboards, Panels & Visualizations]
This is a just a quality of life idea, but JSON is not really human-readable, and YAML is rising in popularity as a more readable alternative.

For scenarios where it is necessary to write, edit or debug the panel or dashboards, it can be cumbersome to do so with JSON. It could be nice to have a button that switches between JSON and YAML when viewing the dashboard or panel structure.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82630. Feel free to continue the discussion around this there. Thank you!
